src/app/markdown/markdown.component.scss

```diff
@@ -0,0 +1,70 @@
+@import "../../assets/stylesheets/_base";
+
+[contenteditable]:focus {
+    outline: 0px solid transparent;
+}
+
+.tab-btn {
+  cursor: pointer;
+}
+
+
+.editor-container {
+    background-color: $color-pf-black-200;
+
```

Comment:
remove the empty line to show that the next class is part of `.editor-container`